common/dependabot-common.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "bundler", ">= 1.16", "< 3.0.0"
   spec.add_dependency "docker_registry2", "~> 1.7", ">= 1.7.1"
   spec.add_dependency "excon", "~> 0.66"
+  spec.add_dependency "faraday", "~> 0.16"
```

Comment:
v0.16 has been/will be yanked - see https://github.com/lostisland/faraday/releases/tag/v0.17.0